<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="4">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,3">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string/>
           </property>
           <widget class="QPushButton" name="m_btnOk">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>140</y>
              <width>81</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>确定</string>
            </property>
           </widget>
           <widget class="QPushButton" name="m_btnQuit">
            <property name="geometry">
             <rect>
              <x>153</x>
              <y>140</y>
              <width>81</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>取消</string>
            </property>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>52</x>
              <y>32</y>
              <width>181</width>
              <height>80</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>卡号: </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="m_editCardNum"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>工号: </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="m_editJobNum"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>姓名: </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="m_editName"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string/>
           </property>
           <widget class="QPushButton" name="m_btnPicPath">
            <property name="geometry">
             <rect>
              <x>49</x>
              <y>147</y>
              <width>75</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>浏览</string>
            </property>
           </widget>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>20</y>
              <width>111</width>
              <height>111</height>
             </rect>
            </property>
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="m_lbPic">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
